internal/subshell/sscommon/sscommon.go

```diff
@@ -127,7 +127,7 @@ func runWithCmd(env []string, name string, args ...string) error {
 	case ".bat":
 		// No action required
 	case ".ps1":
-		args = append([]string{"-file", name}, args...)
+		args = append([]string{"-executionpolicy", "bypass", "-file", name}, args...)
```

Comment:
I wonder if we've worked through the potential implications of this... some customers might have locked down environments and passing this flag could cause execution to fail. I remember we had a discussion about this during stand but I can't recall all of the details.

Reply:
We spoke during stand that we'd like to have Mark test this in a brand-new, unprivileged environment. If that fails, we can undo all of this. In my testing, it seems to work, but I haven't tried all configurations.

Reply:
I do wonder if that will catch potential issues with more complex locked-down environment, but I guess we will see if people complain.